ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -532,6 +532,9 @@ hardware_interface::return_type URPositionHardwareInterface::read()
 
 hardware_interface::return_type URPositionHardwareInterface::write()
 {
+  // If there is no interpreting program running on the robot, we do not want to send anything.
+  // TODO: We would still like to disable the controllers requiring a writable interface. In ROS1
```

Comment:
I think linter give here error because it is expected following line: 
```suggestion
  // TODO(anyone): We would still like to disable the controllers requiring a writable interface. In ROS1
```